### PR TITLE
English translation fixes

### PR DESCRIPTION
**Pull request by xen-42**
Fixed some wrong pronouns, missing determinants, replaced guillemets with quotation marks, improved some wording.

### CODE REVIEW

**Review by hearth1an — COMMENTED** [review]
I'm sure it's Eye of the universe in game, not Universe

**Comment by hearth1an** [comment]
> I'm sure it's Eye of the universe in game, not Universe

nah, you're right

**Comment by hearth1an** [comment]
> I'm sure it's Eye of the universe in game, not Universe

nah, you're right

File: translations/english.json [hunk 1]
```diff
@@ -192,35 +192,35 @@
...
         "Seriously, how could an anglerfish just disappear?": "Seriously, how could an anglerfish just disappear?",
-        "Hornfels said that anglerfish «Ernesto» disappeared from exhibit.": "Hornfels said that anglerfish «Ernesto» disappeared from exhibit.",
+        "Hornfels said that anglerfish «Ernesto» disappeared from exhibit.": "Hornfels said that the anglerfish \"Ernesto\" disappeared from its exhibit.",
         "Ernesto found": "Ernesto found",
         "I found Ernesto": "I found Ernesto",
-        "I found Slate with Ernesto. He's cooking him on campfire.": "I found Slate with Ernesto. He's cooking him on campfire.",
-        "Slate is just fed up with marshmallow, that's why he decided to cook Ernesto. He said it will be ready in 30 minutes. Intervention isn't required because the loop will reset before that happens.": "Slate is just fed up with marshmallow, that's why he decided to cook Ernesto. He said it will be ready in 30 minutes. Intervention isn't required because the loop will reset before that happens."
+        "I found Slate with Ernesto. He's cooking him on campfire.": "I found Slate with Ernesto. They're cooking it on a campfire.",
```
Comment (hearth1an) [inline translations/english.json:222]: They're? Slate is cooking Ernesto, no?

File: translations/english.json [hunk 2]
```diff
@@ -192,35 +192,35 @@
...
+        "Slate is just fed up with marshmallow, that's why he decided to cook Ernesto. He said it will be ready in 30 minutes. Intervention isn't required because the loop will reset before that happens.": "Slate is fed up with marshmallow and decided to cook Ernesto. They said it will be ready in 30 minutes. Intervention isn't required because the loop will reset before that happens."
```
Comment (hearth1an) [inline translations/english.json:223]: Same here. Why they if Slate said it will be ready in 30 min

File: translations/english.json [hunk 3]
```diff
@@ -1,115 +1,115 @@
...
+    "So you decided to eat <![CDATA[<color=orange>Ernesto</color>]]>?! Hornfels said <![CDATA[<color=orange>Ernesto</color>]]> is a friend!": "So you decided to eat <![CDATA[<color=orange>Ernesto</color>]]>?! Hornfels said <![CDATA[<color=orange>Ernesto</color>]]> was their friend!",
+    "Anglerfish is still a fish. Whatever he calls it, is almost done cooking.<![CDATA[<Pause=1>]]>.<![CDATA[<Pause=1>]]>.<![CDATA[<Pause=1>]]> or not. I think it will be prepared in like <![CDATA[<i>30 minutes</i>]]>. Why don't you fly near <![CDATA[<color=orange>Dark Bramble</color>]]> and just get a new one for Hornfels?": "An anglerfish is still a fish. Whatever they call it, it's almost done cooking.<![CDATA[<Pause=1>]]>.<![CDATA[<Pause=1>]]>.<![CDATA[<Pause=1>]]> or not. I think it will be prepared in like, <![CDATA[<i>30 minutes</i>]]>. Why don't you fly near <![CDATA[<color=orange>Dark Bramble</color>]]> and just get a new one for Hornfels?",
```
Comment (hearth1an) [inline translations/english.json:100]: Why they if it's just Slate calling angler Ernesto

File: translations/english.json (hunk 1 above)
Comment (xen-42) [reply translations/english.json:222]: Slate is a they, the Hearthians have a single gender and use they as a pronoun in English.

Comment (hearth1an) [reply translations/english.json:222]: > Slate is a they, the Hearthians have a single gender and use they as a pronoun in English.

what? is there anything in game about it?

**Pull request merged**